chart/kubecop/values.yaml

```diff
@@ -30,6 +30,9 @@ kubecop:
   alertmanager:
     enabled: false
     endpoints: "localhost:9093"
+  httpEndpoint:
+    enabled: false
+    url: "http://synchronizer.kubescape.svc.cluster.local/apis/v1/kubescape.io/v1/RuntimeAlerts"
```

Comment:
Why it is http? we don't encrypt the traffic between our micro services?

Reply:
no we don't

Reply:
Very bad, Very bad, Very bad, Very Very Very Very Very Very bad.